plotter/invertedaxis_example_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter_test
+
+import (
+	"image/color"
+	"log"
+	"math"
+	"testing"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/internal/cmpimg"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg"
+)
+
+// Example_invertedScale shows how to create a plot with an inverted axes.
```

Comment:
This comment will not be rendered. If you want commentary in an example, it must be in the body. In that case you don't need to use godoc phrasing, just normal prose is OK.

Reply:
I don't write too many Test Examples - Its just means I need more practice

I moved the comment into the body as my intent was to provide more prose on how to exactly use it. gonum/plot has a particular cadence/style/feel when compared to other plotting libraries - I feel that more hints as to design patterns should make first time users get moving a bit faster.

Reply:
Yeah, no worries. You're doing good work and sometime the vagaries of godoc rendering a difficult (I usually have a godoc server running and check what the docs look like to make sure everything is good, particularly when examples are part of the docs.).